programmers/난이도별/level01.2016년/rockmiin.py

```diff
@@ -0,0 +1,35 @@
+def solution(a, b):
+    day = ['THU', 'FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED']
+    mon_31 = [1, 3, 5, 7, 8, 10, 12]
+    mon_30 = [4, 6, 9, 11]
+    sum = 0
+
+    for i in range(1, a):
+        if i in mon_31:
+            sum += 31
+        elif i in mon_30:
+            sum += 30
+        else:
+            sum += 29
+    sum += b
```

Comment:
```suggestion
    month = { 1: 31, 2: 29, 3: 31: 4: 30, 5: 31, 6: 30, 7: 31, 8: 31, 9: 30, 10: 31, 11: 30, 12: 31}
    sum = b
    for i in range(1, a):
          sum += month[i]
```
딕셔너리를 사용하면 조금 더 직관적으로 풀이할 수 있어요!